### ISSUE TITLE
Cannot order complex repeatable items

### ISSUE THREAD

**Issue by lorenzomigliorero** [strapi/strapi#5180]
**Describe the bug**

I had a dynamic zone `Lorem` with a single component `bar`.
Component `bar` contains one field, a repeatable `foo` component.

![image](https://user-images.githubusercontent.com/4406419/73761586-59196580-476f-11ea-87a7-b8a90eeb1a27.png)

This content-type causes strange behavior when you try to sorting it.
If you change the order of a `bar` component with another one that contains a different numbers of `foo` components, an error occours:

`TypeError: Cannot read property 'isOpen' of undefined`

**Steps to reproduce the behavior**
1. Create a component `foo`
2. Create a component `bar` that have as unique field a repeatable component `foo`
3. Create a dynamic zone and add the new `bar` component
4. Enter in the dynamic zone, and add a `bar` component
5. Open it and add a single `foo` entry
6. Add another `bar` component and add two `foo`entries
7. Try to order the last `bar` component as first

**Screenshots**

![Screenflick Movie 14](https://user-images.githubusercontent.com/4406419/73761483-371fe300-476f-11ea-9732-b8f009b0d62e.gif)

**System**
- Strapi version: v3.0.0-beta.18.7
- Database: SQLite
- Operating system: MacOs

**Comment by lauriejim** [comment]
Hello! I tried to reproduce your issue and it works on my side.
Here is a short video of what I did - https://www.loom.com/share/34ba0fda1a5045aa8ddcba53ad6f7438
Normally the data schema is respected - Let me know f something is missing.

**Comment by lorenzomigliorero** [comment]
You are right.
I tried on two different Strapi instances, but the error remains.

I record another video, more complete than the previous one: https://www.loom.com/share/09ef010885fd43f0961494cddafd7590

After investigation, i notice that if you try to swap the order between items that contain the same numbers of entries, it's works as expected.

Let me know how can i help you.

**Comment by lauriejim** [comment]
What is the browser you are using?

**Comment by lorenzomigliorero** [comment]
Google Chrome 79.0.3945.130
Node version 10.13.0

**Comment by lorenzomigliorero** [comment]
Ok, i made a mistake.
This bug was fixed in v3.0.0-beta.18.7, i updated only the deps without rebuild the admin.

But...

I got confused by the **poweredby** label in the admin panel.
It's not related to the admin panel, but to the package.json.
If you change the package.json without rebuild the admin, **poweredby**  is updated, but the admin remains to the old version.

This behaviour is a bit confuse, can create a lot of unuseful issues, like this one.

Is it possibile to insert the version in the build phase?
